spec/requests/checkouts_spec.rb

```diff
@@ -48,29 +111,43 @@
   end
 
   describe "POST /create" do
-    context "with valid parameters" do
-      it "creates a new Checkout" do
-        expect {
+    context "json request" do
+      context "with valid parameters" do
+        it "creates a new Checkout" do
+          expect {
+            post checkouts_url, params: { checkout: valid_attributes, format: :json }
+          }.to change(Checkout, :count).by(1)
+        end
+
+        it "redirects to the created checkout" do
```

Comment:
This predates this PR but would be good to fix.
```suggestion
        it "returns created status" do
```